rpc/src/server/mod.rs

```diff
@@ -210,6 +214,19 @@ pub fn spawn_server(
         }))
 }
 
+/// Normalizes the request path
+fn normalize_path(original_path: &str) -> Option<String> {
```

Comment:
@tizoc 
Could you, please, add here some easy unit-test with some corner cases?
Because from description "Normalizes the request path" it is not very obvious, what it is doing :)

Reply:
@bkontur yes, but first I have to understand exactly what hyper and `PathTree` do (and don't do) with the path to understand what needs to be done here (and what doesn't). The current fix is a quick hack to fix multiple `/`s in the path. It is also important that the normalization behaves the same as what Octez does.

Reply:
Or to put it another way, the actual solution is probably much more complicated than this and requires some research, this one here is not the final solution.